sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -11,7 +15,7 @@
             v-model="legalName"
             data-cy="legalName"
             @keyup.enter="searchInstitutions"
-            :rules="[(v) => !!v || 'Legal name required']"
+            hide-details
```

Comment:
as discussed if you add `hide-details` you wont be able to see validation below each fields